lib/Tests/Fhp/Segment/HICAZTest.php

```diff
@@ -0,0 +1,74 @@
+                            '<BkToCstmrAcctRpt><GrpHdr><MsgId>camt52_20131118101510__ONLINEBA</MsgId>' .
+                            '<CreDtTm>2013-11-18T10:15:10+01:00</CreDtTm><MsgPgntn><PgNb>1</PgNb><LastPgInd>true</LastPgInd></MsgPgntn></GrpHdr>' .
+                            '<Rpt><Id>camt052_ONLINEBA</Id>' .
+                            '<Ntry><Sts>BOOK</Sts></Ntry>' .
+                            '<Ntry><Sts>BOOK</Sts></Ntry>' .
+                            '<Ntry><Sts>BOOK</Sts></Ntry>' .
+                            '<Ntry><Sts>BOOK</Sts></Ntry>' .
+                            '</Rpt></BkToCstmrAcctRpt></Document>';
+    private const SAMPLE_XML_DOC3 = '<?xml version="1.0" encoding="UTF-8"?>' .
+                            '<Document xmlns="urn:iso:std:iso:20022:tech:xsd:camt.052.001.02" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ' .
+                            'xsi:schemaLocation="urn:iso:std:iso:20022:tech:xsd:camt.052.001.02 camt.052.001.02.xsd">' .
+                            '<BkToCstmrAcctRpt><GrpHdr><MsgId>camt52_20131118101510__ONLINEBA</MsgId>' .
+                            '<CreDtTm>2013-11-18T10:15:10+01:00</CreDtTm><MsgPgntn><PgNb>1</PgNb><LastPgInd>true</LastPgInd></MsgPgntn></GrpHdr>' .
+                            '<Rpt><Id>camt052_ONLINEBA</Id>' .
+                            '<Ntry><Sts>PDNG</Sts></Ntry>' .
+                            '</Rpt></BkToCstmrAcctRpt></Document>';
+
+    public function testHICAZparse()
+    {
+        // First example: two  XMLs  seperated by ":" - both are gebuchteUmsaetze
+        $hicaz1 = HICAZv1::parse(static::HICAZ_TEST_START .
+                                '@' . strlen(static::SAMPLE_XML_DOC1) . '@' .
+                                static::SAMPLE_XML_DOC1 .
+                                ':' .
+                                '@' . strlen(static::SAMPLE_XML_DOC2) . '@' .
+                                static::SAMPLE_XML_DOC2 .
+                                "'");
+
+        $this->assertEquals([static::SAMPLE_XML_DOC1, static::SAMPLE_XML_DOC2],
+                                $hicaz1->gebuchteUmsaetze->GetData());
+
+        // Second example: two areas  seperated by +, first area has a group of two XMLs seperated by :
+
+        $hicaz2 = HICAZv1::parse(static::HICAZ_TEST_START .
+                                '@' . strlen(static::SAMPLE_XML_DOC1) . '@' .
+                                static::SAMPLE_XML_DOC1 .
+                                ':@' . strlen(static::SAMPLE_XML_DOC2) . '@' .
+                                static::SAMPLE_XML_DOC2 .
+                                '+@' . strlen(static::SAMPLE_XML_DOC3) . '@' .
+                                static::SAMPLE_XML_DOC3 .
+                                "'");
+        $this->assertEquals([static::SAMPLE_XML_DOC1, static::SAMPLE_XML_DOC2],
+                                $hicaz2->gebuchteUmsaetze->GetData());
```

Comment:
GetData() doesn't exist (with capital G). I think the new getGebuchteUmsaetze() getter should be used here instead.

Reply:
done